### PR TITLE
Adding exos_command cli_conf module

### PR DESCRIPTION
**Pull request by rdvencioneck**
##### SUMMARY
This PR adds suport for running "read-only" commands on Extreme OS (EXOS)

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
exos_command

### CODE REVIEW

File: lib/ansible/module_utils/network/exos/exos.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    if hasattr(module, '_exos_capabilities'):
+        return module._exos_capabilities
+
+    capabilities = Connection(module._socket_path).get_capabilities()
+    module._exos_capabilities = json.loads(capabilities)
+
+    return module._exos_capabilities
+
+
+def check_args(module, warnings):
```
Comment (ganeshrn) [inline lib/ansible/module_utils/network/exos/exos.py]: This function is not required.

File: lib/ansible/modules/network/exos/exos_command.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: exos_command
+version_added: "2.6"
+author: "Rafael D. Vencioneck (@rdvencioneck)"
+short_description: Run commands on remote devices running EXOS
```
Comment (ganeshrn) [inline lib/ansible/modules/network/exos/exos_command.py]: Please include the company name in the short_description to make it easier for people to find the modules
See http://docs.ansible.com/ansible/latest/modules/list_of_network_modules.html

File: lib/ansible/modules/network/exos/exos_command.py [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        the number of retries has expired.
+    required: true
+  wait_for:
+    description:
+      - List of conditions to evaluate against the output of the
+        command. The task will wait for each condition to be true
+        before moving forward. If the conditional is not true
+        within the configured number of retries, the task fails.
+        See examples.
+    required: false
```
Comment (ganeshrn) [inline lib/ansible/modules/network/exos/exos_command.py]: required: false lines can be removed, as that's the default

File: lib/ansible/modules/network/exos/exos_command.py [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+    default: null
+    aliases: ['waitfor']
```
Comment (ganeshrn) [inline lib/ansible/modules/network/exos/exos_command.py]: `waitfor` is an old name and not required to be added to new modules. This line can be removed.

File: lib/ansible/modules/network/exos/exos_command.py [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+    return commands
+
+
+def main():
+    """main entry point for module execution
+    """
+    argument_spec = dict(
+        commands=dict(type='list', required=True),
+
+        wait_for=dict(type='list', aliases=['waitfor']),
```
Comment (ganeshrn) [inline lib/ansible/modules/network/exos/exos_command.py]: `aliases` can be removed

**Comment by rdvencioneck** [comment]
@ganeshrn thank you for your review! I'm committing the requested changes right now.

**Review by ganeshrn — APPROVED** [review]
Please update the new files added in this PR https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml along with your GitHub handle

**Pull request merged**